frontend/package.json

```diff
@@ -91,21 +103,5 @@
       "prettier --config .prettierrc --write",
       "eslint --fix"
     ]
-  },
-  "jest": {
-    "collectCoverageFrom": [
```

Comment:
where did it go? I dont see it in config filr

Reply:
collect coverage from was moved to jest.config.js